README.md

```diff
@@ -19,7 +19,9 @@ cargo add eigensdk --features full
 - [eigen-metrics](https://github.com/Layr-Labs/eigensdk-rs/tree/main/crates/metrics) - performance, rpc and economic metrics
 - [eigen-services](https://github.com/Layr-Labs/eigensdk-rs/tree/main/crates/services) - Spawn tokio services for operators info, bls aggregation
 - [eigen-types](https://github.com/Layr-Labs/eigensdk-rs/tree/main/crates/types) - Common types
-- [eigen-utils](https://github.com/Layr-Labs/eigensdk-rs/tree/main/crates/utils) - Publicly exportable `mainnet rewards-v2 v0.5.4` compatible alloy bindings.
+- [eigen-utils](https://github.com/Layr-Labs/eigensdk-rs/tree/main/crates/utils) 
+    - `rewardsv2` - Publicly exportable `mainnet rewards-v2 v0.5.4` compatible alloy bindings.
+    - `slashing` - Publicly exportable alloy bindings compatible with this [commit](https://github.com/layr-labs/eigenlayer-middleware/tree/848032a7e22b2e6a2ab7835a45e9b39bb16d41a5).
```

Comment:
I think we can link the v1.1.1 release here

Reply:
👍